src/js/fc.js

```diff
@@ -68,6 +68,8 @@ var DEFAULT;
 var DEFAULT_PIDS;
 
 var FC = {
+    MOTOR_OUTPUT_ORDER: [],
```

Comment:
You have been fast! The idea was to move all the vars inside the FC object but you have been the first :)
Ok, I will move the others when merged this PR.

Reply:
Oh i thought that's what you wanted me to do lol with this variable lol

Reply:
No problem, is a way to remember that we need to do the same for all the others :)